src/rust/include/librustzcash.h

```diff
@@ -104,14 +104,15 @@ extern "C" {
         unsigned char *result
     );
 
-    /// Compute [sk] [8] P for some 32-byte
-    /// point P, and 32-byte Fs. If P or sk
-    /// are invalid, returns false. Otherwise,
+    /// Compute KDF^Sapling(KA^Agree(sk, P), ephemeral_key).
+    ///
+    /// If P or sk are invalid, returns false. Otherwise,
     /// the result is written to the 32-byte
     /// `result` buffer.
```

Comment:
```suggestion
    /// P and sk must point to 32-byte buffers. If P does not
    /// represent a Jubjub point or sk does not represent a
    /// canonical Jubjub scalar, this function returns false.
    /// Otherwise, it writes the result to the 32-byte `result`
    /// buffer and returns true.
```